edk2toolext/environment/extdeptypes/az_cli_universal_dependency.py

```diff
@@ -147,11 +147,24 @@ def _attempt_universal_install(self, install_dir: str) -> None:
         ret = RunCmd(cmd[0], " ".join(cmd[1:]), outstream=results, environ=e)
         if ret != 0:
             results.seek(0)
-            raise Exception(f"\nCommand \"{' '.join(cmd)}\" failed with {ret}.\n\n" f"{results.getvalue()}")
+            raise Exception(f'\nCommand "{" ".join(cmd)}" failed with {ret}.\n\n{results.getvalue()}')
 
         # az tool returns json data that includes the downloaded version
         # lets check it to double confirm
-        result_data = json.loads(results.getvalue())
+        try:
+            result_data = json.loads(results.getvalue())
+        except Exception as e:
+            logging.info("Failed to parse json data from az command output: %s", e)
```

Comment:
nit: Can this use an f-string too?

```suggestion
            logging.info(f"Failed to parse json data from az command output: {e}")
```